test/unit/webdriver/device/keyboard_test.py

```diff
@@ -32,3 +32,27 @@ def test_hide_keyboard(self):
             appium_command('/session/1234567890/appium/device/hide_keyboard')
         )
         assert isinstance(driver.hide_keyboard(), WebDriver)
+
+    @httpretty.activate
+    def test_press_keycode(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.POST,
+            appium_command('/session/1234567890/appium/device/press_keycode'),
+            body='{"value": "86"}'
+        )
+        driver.press_keycode('86')
+        d = get_httpretty_request_body((httpretty.last_request()))
+        assert d['keycode'] == '86'
```

Comment:
Please add checks for metastate and flags as arguments.

https://github.com/appium/python-client/blob/master/appium/webdriver/extensions/keyboard.py#L73

And add another test cases for metastate and flags with values.